GameOfLife/Controller/GameManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Management of the game
+    /// </summary>
+    public class GameManager
+    {
+        /// <summary>
+        /// Starts the game
+        /// </summary>
+        public void Play()
+        {
+            UserInterface userInterface = new UserInterface();
+            GameView gameView = new GameView();
+            UserInterface.WelcomeScreen();
+            
+            int column = userInterface.ValidateUserInput(UserInterface.inputColomnMessage, GameParameters.minInputValue, GameParameters.maxInputValue);
+            int row = userInterface.ValidateUserInput(UserInterface.inputRowMessage, GameParameters.minInputValue, GameParameters.maxInputValue);
+            Game game = new Game(column, row);
+
+            gameView.View(game.gameField);
+
+            while (true)
+            {
+                game.GetNextGeneration();
+                gameView.View(game.gameField);
+                Thread.Sleep(1000);
+            }
+
```

Comment:
Please remove empty lines. There should be empty line only after if, for, while or other code block if not followed by closing brace of other block.

Reply:
+1

Reply:
Changes have been made.